app/scripts/lib/accounts/BalancesController.ts

```diff
@@ -249,12 +264,35 @@ export class BalancesController extends BaseController<
     const partialState: BalancesControllerState = { balances: {} };
 
```

Comment:
This method will be updated once we have the network picker. The scope will adapt based on the network selected.